### ISSUE TITLE
Varia & Varia Child Themes: Accessibility related issues in Comments

### ISSUE THREAD

**Issue by itowhid06** [rtCamp/themes#23]
## Steps to replicate

1. Go to any posts with comments.
2. Inspect any comment author's name in Devtools.
3. See the author's name is wrapped in a `<b>` tag.

## Result

![image](https://user-images.githubusercontent.com/12055657/82238965-e5841880-9959-11ea-99f0-2439f328af5f.png)

As evident in the above screenshot the author's name is wrapped in a `<b>` tag. This is hapenning because, though there is a Comment Walker class named as `Varia_Walker_Comment` it is not used in `wp_list_comments` function. As a result the aforementioned function is using `Walker_Comment` Walker class from WordPress Core, which is producing those `<b>` tags. As mentioned in [https://core.trac.wordpress.org/ticket/50202](url), `<b>` tags should be avoided in favor of `<strong>` tags for Accessibility reasons. Note that, there is also an extra badge printing after the comment author's name when `Varia_Walker_Comment` walker is used.

## Expected

 The author's name in a comment should be wrapped in a `<strong>` tag as evident in many core themes like **Twenty Nineteen** & **Twenty Twenty** themes.  Also Varia theme should use `Varia_Walker_Comment` walker.
